unlock-app/src/components/interface/Receipts/elements/ReceiptBox.tsx

```diff
@@ -75,6 +81,7 @@ export const ReceiptBox = ({ lockAddress, hash, network }: ReceiptBoxProps) => {
     lockAddress,
     hash,
     network,
+    tokenId,
```

Comment:
One consideration is that a receipt can have multiple tokenIds, for example when someone purchases multiple tickets in a single transaction. In that case we should not backfill the receipt details because we would not know which of the token ids matches a specific receipt.

Reply:
added check on here https://github.com/unlock-protocol/unlock/pull/11961/files#diff-1a65a5d44afd090b7b7a1f6a33a13d07a254a2e7248729c8e04159986188d820R17-R22